deploy.json

```diff
@@ -1,5 +1,5 @@
 {
-  "defaultStacks" : ["flexible"],
+  "defaultStacks" : ["flexible", "flexible-secondary"],
```

Comment:
This will work for the `lambda-upload` but not `lamda-update` as below the functionNames are hardcoded to start `flexible-`.